compiler/damlc/tests/src/DA/Test/DataDependencies.hs

```diff
@@ -27,7 +26,6 @@ import SdkVersion
 
 main :: IO ()
 main = do
-    setEnv "TASTY_NUM_THREADS" "1" True
```

Comment:
given that you report a 2x speedup how much do we gain by setting this to 2 and 4? We should also change the resources Bazel allocates for this either if we don’t fix the number of CPUs here by marking it exclusive or via `    tags = ["cpu:2"],`.

Reply:
I just tried this now on my laptop:

- Setting TASTY_NUM_THREADS to 1: 229.0s
- Setting TASTY_NUM_THREADS to 2: 142.3s
- Setting TASTY_NUM_THREADS to 3: 103.9s
- Setting TASTY_NUM_THREADS to 4: 96.5s
- Not setting TASTY_NUM_THREADS: 134.5s

I'm going to set it to 3. Should I change anything else (e.g. in Bazel)?

Reply:
sounds good, then let’s add `cpu:3` to the tags to make sure we don’t schedule too much.